web/src/components/Settings/WorkspaceSection.tsx

```diff
@@ -18,6 +20,7 @@ const WorkspaceSection = () => {
     workspaceSettingStore.getWorkspaceSettingByKey(WorkspaceSettingKey.GENERAL)?.generalSetting || {},
   );
   const [workspaceGeneralSetting, setWorkspaceGeneralSetting] = useState<WorkspaceGeneralSetting>(originalSetting);
+  const [identityProviderList, setIdentityProviderList] = useState<IdentityProvider[]>([]);
```

Comment:
Can this be a simple variable? I dont believe there is and need to have it's state tracked.